ios/Fabric/RNCPagerViewComponentView.mm

```diff
@@ -398,6 +406,29 @@ + (ComponentDescriptorProvider)componentDescriptorProvider
 }
 
 
+- (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecognizeSimultaneouslyWithGestureRecognizer:(UIGestureRecognizer *)otherGestureRecognizer {
+
+    // Recognize simultaneously only if the other gesture is RN Screen's pan gesture (one that is used to perform fullScreenGestureEnabled)
+    if (gestureRecognizer == self.panGestureRecognizer && [NSStringFromClass([otherGestureRecognizer class]) isEqual: @"RNSPanGestureRecognizer"]) {
```

Comment:
`RNSPanGestureRecognizer` is the name of the pan gesture used by [react-native-screens](https://github.com/software-mansion/react-native-screens/blob/4438732600e2d47105c4ffd9e62b33ef8723430e/ios/RNSScreenStack.mm#L82). I couldn't find a better approach than this, as [discussed here](https://github.com/callstack/react-native-pager-view/pull/665#discussion_r1038906896)